recipe/meta.yaml

```
@@ -58,9 +60,9 @@ requirements:
     - mlflow >=2.1.0,<2.4
     - sentencepiece >=0.1.95,<0.2
     - tensorflow >=2.9,<3,!=2.12.0
-    - tokenizers>=0.10,<1
+    - tokenizers >=0.10,<1
     - torchdata >=0.4,<1
-    - transformers >=4.29.2,<5
+    - transformers >=4.32.1,<5
```

```suggestion
    - transformers >=4.32.1,<5
    - vllm >=0.2.1.post1,<1
```
https://github.com/snowflakedb/snowflake-ml-python/blob/1.0.12/snowflake/ml/requirements.bzl#L15C10-L15C30